tests/cypress/e2e/hedy_page/run_code_button.cy.js

```diff
@@ -4,6 +4,7 @@ describe('Is able to run code', () => {
     it('Passes', () => {
       goToHedyPage();
 
+      cy.get('#editor').type("print Hallo!'\n");
```

Comment:
I've seen this not work well in the past. In one of my PRs I changed this to something that seems to work better:

```suggestion
      cy.get('.cm-content').click();
      cy.focused().type("print Hallo!'\n");
```

Reply:
Ah great idea! I added the same change to the other Cypress test I updated!